shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -109,7 +120,20 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
     AlluxioURI path = new AlluxioURI(args[0]);
     int replication = FileSystemShellUtils.getIntArg(cl, REPLICATION_OPTION, DEFAULT_REPLICATION);
-    distributedLoad(path, replication);
+
+    if (!cl.hasOption('f')) {
+      distributedLoad(path, replication);
+    } else {
+      FileInStream is = mFileSystem.openFile(path);
```

Comment:
did we decide to use local file system for the index file path? if  that is the case, perhaps use `BufferedReader` ? something like `BufferedReader br = new BufferedReader(path);`